### PR TITLE
TSD Show wind & endurance Logic Correction

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mfdConstants.h [hunk 1]
```diff
@@ -165,14 +165,13 @@
...
 #define MFD_IND_TSD_ROOT_SHOW_WPT_DATA_CURRTE 0
-#define MFD_IND_TSD_ROOT_SHOW_WIND 8
-#define MFD_IND_TSD_ROOT_SHOW_ENDR 9
 
 #define MFD_TEXT_IND_TSD_ROOT_ENDR 0
 #define MFD_TEXT_IND_TSD_ROOT_WPDEST 1
 #define MFD_TEXT_IND_TSD_ROOT_GROUNDSPEED 2
 #define MFD_TEXT_IND_TSD_ROOT_WPDIST 3
 #define MFD_TEXT_IND_TSD_ROOT_WPETA 4
+#define MFD_TEXT_IND_TSD_ROOT_Wind 5
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mfdConstants.h:174]: WIND

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getwindDirVel.sqf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/* ----------------------------------------------------------------------------
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getwindDirVel.sqf:1]: The function name needs correcting to getWindDirVel with a capital W

File: @AH-64D Apache Longbow/Addons/fza_ah64_systems/cfgVehicles/hitPoints.hpp [hunk 3]
```diff
@@ -338,7 +338,7 @@ class HitPoints
         radius = HIT_RADIUS_01;
         minimalHit = MIN_HIT;
         explosionShielding = 0.80;
-        name = "hitvrotor";
+        name = "";
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_systems/cfgVehicles/hitPoints.hpp:341]: why was this removed? it's not removed from hithrotor

File: @AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/core/fn_coreUpdate.sqf [hunk 4]
```diff
@@ -18,7 +18,7 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli"];
 
-private _deltaTime = ((["systems_deltaTime"] call BIS_fnc_deltaTime) min 1/30);
+private _deltaTime = ["systems_deltaTime"] call BIS_fnc_deltaTime;
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/core/fn_coreUpdate.sqf:21]: Not related, but agree with it

**Review by BradMick — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_systems/cfgVehicles/hitPoints.hpp (hunk 3 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_systems/cfgVehicles/hitPoints.hpp:341]: Accidental

**Review by BradMick — APPROVED** [review]
